src/app/modules/main/profile_section/profile/models/profile_save_data.nim

```diff
@@ -0,0 +1,50 @@
+import json, strutils, sequtils
+
+include app_service/common/json_utils
+include app_service/common/utils
+
+import app_service/service/profile/dto/profile_showcase_preferences
+
+type ProfileShowcaseSaveEntry* = ref object of RootObj
+  key*: string
+  showcaseVisibility*: ProfileShowcaseVisibility
```

Comment:
```suggestion
  visibility*: ProfileShowcaseVisibility
```

Reply:
I would like to stick with [showcaseVisibility](https://github.com/status-im/status-desktop/issues/13688#issuecomment-1960210809)